artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/v11/StompFrameV11.java

```diff
@@ -66,7 +66,7 @@ public ActiveMQBuffer toActiveMQBuffer() throws Exception
             head.append(h.getEncodedValue());
             head.append(Stomp.NEWLINE);
          }
-         if (bytesBody != null && bytesBody.length > 0)
+         if (bytesBody != null && bytesBody.length > 0 && !hasHeader(Stomp.Headers.CONTENT_LENGTH))
```

Comment:
I don't think there's any way to know. But then again there probably isn't any way to know what might happen on client side if there are duplicate or multiple content-length headers.